tests/installation/start_install.pm

```diff
@@ -134,4 +134,10 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    $self->save_upload_y2logs if check_var('UPGRADE', '1');
+    $self->SUPER::post_fail_hook;
```

Comment:
This will lead to upload two times y2log.

Reply:
Done, set_var('Y2LOGS-UPLOADED',1) to fixt this.